lib/msal-core/src/cache/AuthCache.ts

```diff
@@ -0,0 +1,189 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { Constants, CacheKeys } from "../utils/Constants";
+import { AccessTokenCacheItem } from "./AccessTokenCacheItem";
+import { CacheLocation } from "../Configuration";
+import { BrowserStorage } from "./BrowserStorage";
+
+/**
+ * @hidden
+ */
+export class AuthCache extends BrowserStorage {// Singleton
```

Comment:
Is this a singleton anymore?

Reply:
I think it is, this PR is not intended to make any changes for the existing storage design, just added a layer for separation on data base functions vs library related key generation.